src/Utilities.cs

```diff
@@ -12,6 +12,11 @@ public class Utilities {
         private static FieldInfo fieldSetContractEmployers = AccessTools.Field(typeof(StarSystemDef), "contractEmployerIDs");
         private static FieldInfo fieldSetContractTargets = AccessTools.Field(typeof(StarSystemDef), "contractTargetIDs");
 
+        public static FactionValue GetFactionValueByFactionID(string id)
```

Comment:
I've been using camelCase (humps in the middle) function names rather than PascalCase (first letter capitalized). So `getFactionValueByFactionID`